CRM/CivirulesActions/Participant/AddToZoom.php

```diff
@@ -130,7 +130,15 @@ private function addParticipant($participant, $entityID, $triggerData, $entity)
 			'Content-Type' => 'application/json;charset=UTF-8',
 			'Authorization' => "Bearer $token"
 		])->post($url, $participant);
+		$result = $response->json();
+
+		CRM_Core_Error::debug_var('Zoom addParticipant result', $result);
 
+		// Added the registrant_id and event_id to the log
+		$msg = "Event Id is ".$event['id']. ". ";
+		if(!empty($result['registrant_id'])){
+			$msg = "Registrant Id is ".$result['registrant_id']. ". ";
+		}
```

Comment:
@damo-veda i do not see the $this->logAction call for this event/registrant id information? or it should be appended to success or failure message which are already logged.